pkg/config/v2/type_definition_test.go

```diff
@@ -219,7 +219,7 @@ settings:
 			name:  "wrong data type",
 			given: given{"0x12d4"},
 			expected: expected{
-				errorMessage: "'type' section is not filed with proper values",
+				errorMessage: "failed to parse 'type' section: '' expected a map, got 'int'",
```

Comment:
there is no type section in this error sample... The previous error message completely swallowed the actual parsing error message though, so IMO this is better, even though it produces an equally useless message if the data is way off - rather than just a typo or forgotten sub-field